Source/WebKit/GPUProcess/graphics/WebGPU/RemoteBindGroupLayout.cpp

```diff
@@ -52,7 +52,8 @@ RemoteBindGroupLayout::~RemoteBindGroupLayout() = default;
 
 void RemoteBindGroupLayout::destruct()
 {
-    m_objectHeap->removeObject(m_identifier);
+    if (RefPtr objectHeap = m_objectHeap.ptr())
+        objectHeap->removeObject(m_identifier);
```

Comment:
The theory of WeakRef is that it's not null, and there's no meaningful way to handle null. So this is probably better written without the null check:

`protectedObjectHeap()->removeObject(m_identifier);`